### PR TITLE
Fix math on Windows

### PR DESCRIPTION
**Pull request by bellenot**
- Remove old Windows specific limitations (leftover from ROOT 5)
- Global namespace in front of ROOT::Fit to prevent compiler complaining about ROOT::Fit::ROOT::Fit::ExecutionPolicy
  (maybe we should simply remove ROOT::Fit)

### CODE REVIEW

**Comment by bellenot** [comment]
@amadio I think you approved/commented on the wrong PR... :wink:

File: math/mathcore/test/CMakeLists.txt [hunk 1]
```diff
@@ -74,6 +74,12 @@ ROOT_STANDARD_LIBRARY_PACKAGE(TrackMathCoreUnitDict
                               LINKDEF stress/TrackMathCoreLinkDef.h
                               DEPENDENCIES Core MathCore RIO GenVector)
 
+if(MSVC)
+  add_custom_command(TARGET TrackMathCoreUnitDict POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libTrackMathCoreUnitDict.dll
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libTrackMathCoreUnitDict.dll)
+endif()
+
```
Comment (amadio) [inline math/mathcore/test/CMakeLists.txt]: Nope :-) This is what I am talking about.

Comment (bellenot) [reply math/mathcore/test/CMakeLists.txt]: Oops! right, sorry, and well, if we don't copy it, we have to add `${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>`  in the `PATH` in order to find/load the dll

Comment (amadio) [reply math/mathcore/test/CMakeLists.txt]: That may not be a bad idea, if that's how Windows finds libraries.

**Comment by lmoneta** [comment]
The changes are fine with me. Please merge the PR

 Thank you 

 Lorenzo

**Pull request merged**